addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -558,7 +558,8 @@ export const editorCommands = {
     // Change tags
     setTag(editor, tagName) {
         const restoreCursor = preserveCursor(editor.document);
-        const selectedBlocks = [...new Set(getTraversedNodes(editor.editable).map(closestBlock))];
+        const range = getDeepRange(editor.editable, { correctTripleClick: true });
+        const selectedBlocks = [...new Set(getTraversedNodes(editor.editable, range).map(closestBlock))];
```

Comment:
About the commit message, could you update the commit title to  "[FIX] web_editor: setTag after triple click should not update next line" please ?

Also, you wrote "clickk" instead of "click"  ;-)

Reply:
The code change suggestion and rest other have been applied. thankyou :-)